setup.py

```diff
@@ -276,13 +276,13 @@ def finalize_options(self):
     ])
 
 if (3, 0) < sys.version_info < (3, 7):
-    linux_testenv.extend(['nmslib'])    
-    
+    linux_testenv.extend(['nmslib'])
+
 docs_testenv = linux_testenv + distributed_env + [
     'sphinx',
     'sphinxcontrib-napoleon',
     'plotly',
-    'pattern <= 2.6',
+    'Pattern >= 3.6',
```

Comment:
Is there a reason why we need to specify the version here?

Reply:
To get version with python3 support.

Reply:
Could you please mention that in a code comment? Someone is likely to ask the same question later on.

Reply:
Pushed